File: 250611/특정 문자의 등장 횟수/number-appearances-of-a-particular-character.java
import java.util.*;

public class Main {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        String str = sc.next();
        int eeCnt = 0;
        int ebCnt = 0;
        
        for (int i = 0; i < str.length()-1; i++) {
            if (str.charAt(i)=='e') {
                if (str.charAt(i+1)=='e') {
                    eeCnt++;
                } else if (str.charAt(i+1)=='b') {
                    ebCnt++;
                }
            }
        }
        System.out.print(eeCnt + " " + ebCnt);
    }
}

File: 250911/특정 문자의 등장 횟수/number-appearances-of-a-particular-character.py
s = input()

eeCnt = 0
ebCnt = 0

for i in range(0,len(s)-1) :
    if s[i]=='e' and s[i+1]=='e' :
        eeCnt += 1

    if s[i]=='e' and s[i+1]=='b' :
        ebCnt += 1

print(eeCnt,ebCnt)